pupa/utils/model_ops.py

```diff
@@ -0,0 +1,113 @@
+from django.db import transaction
+from django.db.models import get_models, Model
+from django.contrib.contenttypes.generic import GenericForeignKey
+
+import json
+import logging
+
+logger = logging.getLogger("")
+
+@transaction.commit_on_success
+def merge_model_objects(primary_object, alias_objects=[], keep_old=False):
```

Comment:
right, so i jacked this from django snippets and changed it for my own purposes.

it should be noted that it *doesn't* do the right thing wrt memberships.